cmd/openshift-tests/upgrade.go

```diff
@@ -90,6 +90,9 @@ func initUpgrade(value string) error {
 	if err := json.Unmarshal([]byte(value), &opt); err != nil {
 		return err
 	}
+	if len(opt.ToImage) == 0 {
```

Comment:
General bug we have?

Reply:
This is due to a new subcommand "run-dr" - initUpgrade is being called too and has to return early if no image is specified